src/content/docs/tutorial-diagnose-infra-bottlenecks/investigate-spike-CPU.mdx

```diff
@@ -0,0 +1,148 @@
+---
+title: Investigate a spike in CPU
+metaDescription: Troubleshoot a high CPU process that's leading to system outages.
+---
+
+import cpuSpike from 'images/infrastructure_screenshot-full_cpu.webp'
+
+import hostProcesses from 'images/infrastructure_screenshot-full_process.webp'
+
+import checkLoad from 'images/infrastructure_screenshot-crop_load.webp'
+
+import savedViews from 'images/infrastructure_screenshot-crop_view.webp'
+
+import createView from 'images/infrastructure_screenshot-crop_create_view.webp'
+
+import searchView from 'images/infrastructure_screenshot-crop_search_view.webp'
+
+Now that you understand how to identify failure points in your system, it's time to move to the next layer of the troubleshooting process. Let's say that you've been notified that there's a problem with the underlying infrastructure that supports a team's apps. Building off the skills from [Diagnose app failure with host data](/docs/tutorial-diagnose-infra-bottlenecks/diagnose-app-infra-data), you'll learn one way to compare data so you can make a decision about resource allocation.   
```

Comment:
This is more like a different way to compare data, right? So, diagnose app failures is one way and then this is a second method?